production/grafana-agent-mixin/alerts.libsonnet

```diff
@@ -242,12 +242,12 @@ local _config = config._config;
           {
             alert: 'AgentManagementBadAPIRequests',
             expr: |||
-              100 * sum(rate(agent_remote_config_fetches_total{status_code!~"2.."}[5m])) by (%(group_by_cluster)s)
+              100 * sum(rate(agent_remote_config_fetches_total{status_code="(4|5).."}[10m])) by (%(group_by_cluster)s)
                 /
-              sum(rate(agent_remote_config_fetches_total[5m])) by (%(group_by_cluster)s)
+              sum(rate(agent_remote_config_fetches_total[10m])) by (%(group_by_cluster)s)
                 > 5
             ||| % _config,
-            'for': '10m',
+            'for': '20m',
```

Comment:
AFAIK we don't need to change this when increasing the rate interval (it just means it looks back a longer time frame for each data point, but the alert duration, meaning how much we wait before alerting, is not affected)